package/gluon-config-mode-geo-location/src/de.po.in

```diff
@@ -0,0 +1,54 @@
+msgid ""
+msgstr ""
+#ifdef WITHMAP
+"Project-Id-Version: gluon-config-mode-geo-location-with-map\n"
+#else
+"Project-Id-Version: gluon-config-mode-geo-location\n"
+#endif
+"PO-Revision-Date: 2017-08-22 12:14+0100\n"
+"Last-Translator: Jan-Tarek Butt <tarek@ring0.de>\n"
+"Language-Team: German\n"
+"Language: de\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+msgid ""
+"If you want the location of your node to be displayed on the map, you can "
+"enter its coordinates here. "
+#ifdef WITHMAP
+"If your PC is connected to the internet you may also click on the map displayed below. "
+#endif
+"Please keep in mind setting a location can also enhance the network quality."
+msgstr ""
+"Um deinen Router auf der Karte anzeigen zu können, benötigen wir seine "
+"Koordinaten. Hier hast du die Möglichkeit, "
+"die Koordinaten händisch zu hinterlegen."
+#ifdef WITHMAP
+"Wenn dein Computer mit dem du den Router einrichtest eine Internetverbindung hat, "
+"hast du die Möglichkeit auf der unten angezeigten Karte an die Stelle zu klicken, an der "
+"der Router stehen wird."
+#endif
+"Bitte berücksichtige, dass das Setzen einer Position die Netzwerk Qualität verbessern kann."
+
+msgid "Geo-Location"
+msgstr "Geo-Position"
+
+msgid "Static location"
+msgstr "Manuelle Position"
+
+msgid "Disabled"
+msgstr "Deaktiviert"
+
+msgid "Share your location to see your router on the map"
+msgstr "Position für die Karte freigeben"
```

Comment:
Odd,  but a map might not be the only application receiving that data, so maybe generalize?

Reply:
I didn't change anything there. This is from the master fork.

Reply:
so this should be an issue in the main gluon repository then